src/app/drag-and-drop/basic-droppable.demo.html

```diff
@@ -0,0 +1,176 @@
+<!--
+  ~ Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+
+<h4>Basic Droppable</h4>
+
+<div class="form-group droppable-demo droppable-feature-variant" (change)="activateDemoVariant($event)">
+    <label>with: </label>
+
+    <div class="radio-inline">
+        <input type="radio" id="basic-draggable" name="droppable-variant" checked>
+        <label for="basic-draggable">Basic Draggable</label>
+    </div>
+    <div class="radio-inline">
+        <input type="radio" id="custom-ghost" name="droppable-variant">
+        <label for="custom-ghost">Custom Ghost</label>
+
+    </div>
+    <div class="radio-inline">
+        <input type="radio" id="drag-handle" name="droppable-variant">
+        <label for="drag-handle">Drag Handle</label>
+    </div>
+
+</div>
+
+<div class="clr-example" *ngIf="activeDemoVariant==='basic-draggable'">
+
+    <div class="row">
```

Comment:
I guess this is the old branch, but you'll need to use the new grid when you rebase.

Reply:
Yes, I am expecting substantial changes during rebasing. I will keep your suggestion in mind. 👌